aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -343,7 +342,7 @@ void bgemm<at::BFloat16>(CUDABLAS_BGEMM_ARGTYPES(at::BFloat16)) {
                                    (int) num_batches, rocblas_datatype_f32_r, rocblas_gemm_algo_standard,
                                    0, 0, NULL, NULL));
   #else
-    TORCH_CHECK(false, "BFloat16 bgemm in CUDA requires Ampere or later GPU");
+    TORCH_CHECK(false, "BFloat16 requires CUDA 11");
```

Comment:
I suggest adding the function reference back to clarify where this error comes from:

"CUDA bfloat16 bgemm requires CUDA 11 or later"

Reply:
fixed